src/main/java/com/academy/terai/Controller/AccountResource.java

```diff
@@ -0,0 +1,54 @@
+package com.academy.terai.Controller;
+
+import com.academy.terai.Model.Account;
+import com.academy.terai.Repository.AccountRepository;
+import com.academy.terai.service.AccountService;
+import javassist.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.management.openmbean.KeyAlreadyExistsException;
+import java.util.List;
+
+@RestController
+@RequestMapping("/accounts")
+public class AccountResource {
+    private final AccountService accountService;
+    @Autowired
+    public AccountResource(AccountService accountService){
+        this.accountService = accountService;
+    }
+    @GetMapping(value = "/all")
+    public List<Account> getAllAccounts() {
+        return accountService.findAll();
+    }
+
+    @GetMapping(value = "/{email}")
+    public Account getAccountByEmail(@PathVariable("email") String email) throws NotFoundException {
+        return accountService.findByEmail(email);
+    }
+
+    @GetMapping(value = "/orderByReviewedApplications")
```

Comment:
Endpoints naming is a big topic with many different opinions, however, I'd suggest to think like this:
each / symbol separates different behavior, so, it'd make sense to split order and reviewed applications condition. Not sure if it sounds clear enough, but I'd name it like this:
/order/reviewed
We get a few advantages with such naming, like, short, new ordering endpoint can be added and it'd fit into this naming pattern well, like, order/latest, and so on.